### ISSUE TITLE
Image results lack a focus state

### ISSUE THREAD

**Issue by zackkrida** [WordPress/openverse-frontend#1004]
## Description

The image search endpoint: https://search-staging.openverse.engineering/search/image?q=dogs does not have any focus state on the single image results.

## Expectation

These images should have the same focus state as images in the 'all results' view.
